src/pages/admin/VerifyStaff.jsx

```
@@ -1,93 +1,271 @@
+    await updateDoc(doc(db, 'users', v.applicantUid), {
       verified: false,
-      verificationStatus: "rejected",
+      verificationStatus: 'rejected',
       updatedAt: serverTimestamp(),
     });
+    try {
+      await updateDoc(doc(db, 'verification_requests', v.id), {
+        status: 'rejected',
+        reviewedAt: serverTimestamp(),
+      });
+    } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)